src/components/BackButton/index.js

```diff
@@ -0,0 +1,14 @@
+import Link from 'next/link'
+
+const BackButton = ({ href }) => (
+  <Link href={href}>
+    <a
+      className="govuk-back-link lbh-back-link govuk-!-display-none-print"
```

Comment:
I think you can delete the `govuk-!-display-none-print` as you don't need it at the moment

Reply:
There's a `govuk-!-display-none-print` on the header so I think best to keep it for now - probably need to add it so the 'not live' banner and the phase banner as well.